libs/domain/picking/src/services/adapter/picking-list-default.adapter.ts

```diff
@@ -188,6 +188,7 @@ export class PickingListDefaultAdapter implements PickingListAdapter {
       cartNote: cardNote,
       createdAt: new Date(data.createdAt),
       updatedAt: new Date(data.updatedAt),
+      orderReferences: data.orderReferences,
```

Comment:
This is not needed for default adapter.

Reply:
this will be cover on: https://github.com/spryker/oryx/pull/317